src/cli/changeset/init.ts

```diff
@@ -20,7 +23,7 @@ const getOptionalInput = (name: string) => core.getInput(name) || undefined;
 
   console.log('setting GitHub credentials');
```

Comment:
😰 Do we need this? Seems kinda gross to have to write this to file.

Reply:
I honestly have no clue, I can give it a run without it